cord/client.py

```diff
@@ -783,7 +780,7 @@ def fitted_bounding_boxes(self,
             AuthorisationError: If access to the specified resource is restricted.
             UnknownError: If an error occurs while running interpolation.
         """
-        if len(frames) == 0 or len(video) == 0:
+        if not frames or not video:
```

Comment:
Function `CordClientProject.fitted_bounding_boxes` refactored with the following changes:

- Simplify sequence comparison ([simplify-len-comparison](https://docs.sourcery.ai/refactorings/simplify-len-comparison/))